www/source/blog/2018-05-21-On-Prem-Builder.html.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Introducing Habitat On-Premise Builder
+date: 2018-05-21
+author: Salim Alam
+tags: habitat, builder, depot, on-premise
+category: builder
+classes: body-article
+---
+
+### Introduction
+
+One of the frequent requests that we have had since we introduced the SaaS Habitat Builder last year has been to allow Habitat users to have the same functionality in their on-premise infrastructure. There are several reasons why this can be desirable - for example, having an enterprise policy that prohibits hosting of internal software in the public cloud, or having a completely "air-gapped" network isolation requirement. 
+
+Running Builder on-premise represents an interesting challenge. On the one hand, it is a fairly sophisticated set of scaled out services with a broad range of operational parameters. On the other hand, it is also entirely self-hosted with Habitat itself, and therefore can take advantage of all the Habitat capabilities for deploying, configuring and managing services. So the undertaking was not as daunting as it might have seemed otherwise.
+
+Earlier in the year we rolled out an alpha version of the on-premise Builder Depot to some of Chef's early-adopter partners. We are now happy to make the functionality available more broadly, so that more members of the Habitat community can make use of these capabilities.
+
+### What is Habitat Builder Depot?
+
+The Habitat Builder Depot is the set of services that provide the capability to host Habitat packages. It comes with a Web front end to allow users to log in with a number of different authentication providers, and also hosts a REST API that Habitat clients (i.e., the `hab` binary running from any workstation) can be pointed to.
+
+The set of capabilities that are provided:
+
+* Logging into the on-premise Builder Depot web site
+* Creation of origins, keys, access tokens, etc
+* Invitation of users to origins
+* Upload and download of Habitat packages
+* Promotion and demotion of Habitat packages to channels
+* Normal interactions of the `hab` client with the Builder API
+* Package builds using the `hab` client and Habitat Studio
+* Ability to import core packages from the upstream Habitat Builder
+
+This is a fairly rich sub-set of the SaaS Builder capabilities that are now available on-premise. 
```

Comment:
The "that are now available on-premise" suggests there are some capabilities available today and that these may be a subset of those. Maybe remove that last bit altogether and just say:

```
This is a fairly rich subset of the capabilities available in SaaS Builder today.
```

.. or the like.

Reply:
Yeah, maybe "public" is preferable to "SaaS", to Tasha's point.